### PR TITLE
doc: add control plane documentation for GCP private location configuration

### PR DESCRIPTION
**Pull request by Obero**
image

### CODE REVIEW

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      # GCP project id as returned by GCP CLI:
+      # gcloud projects list
+      project = "<MyProject ID>"
+      # Java configuration (following configuration properties are optional)
+      # System properties (optional)
+      system-properties {
+        "java.net.preferIPv6Addresses" = "true"
+      }
+      # Overwrite JAVA_HOME definition (optional)
+      java-home = "/usr/lib/jvm/zulu17"
```
Comment (bastien-gatling) [inline content/reference/admin/private_locations/configuration/gcp/index.md:69]: I think there is no more suffix in our official images.
```suggestion
      java-home = "/usr/lib/jvm/zulu"
```

@notdryft, you can confirm ?

**Review by bastien-gatling — COMMENTED** [review]
We had some issue with people that copy/paste conf and set some optional conf that could be kept unchanged (like jmvoptions).

Maybe we should comment all the optional conf ?

**Comment by Obero** [comment]
> We had some issue with people that copy/paste conf and set some optional conf that could be kept unchanged (like jmvoptions).
> 
> Maybe we should comment all the optional conf ?

If this is error prone for our users, I agree for this solution :)

@tpetillot, @Isammoc, @stb13579, WDYT ?

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+date: 2023-10-02T15:29:00+00:00
+lastmod: 2023-10-02T15:29:00+00:00
+weight: 22054
+---
+
+## GCP Virtual Machines
+GCP private locations require the control plane to have GCP access rights configured in order to instantiate virtual machines.
+
+### Service account
+Access rights can be set through a service account associated to your control plane.
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/gcp/index.md:14]:
```suggestion
Access rights can be set through a service account associated with your control plane.
```

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        # Storage configuration
+        disk {
+          # Disk size in Gb (mininum 20Gb)
+          sizeGb = 20
+        }
+      }
+      # GCP project id as returned by GCP CLI:
+      # gcloud projects list
+      project = "<MyProject ID>"
+      # Java configuration (following configuration properties are optional)
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/gcp/index.md:63]:
```suggestion
      # Java configuration (the following configuration properties are optional)
```

**Review by stb13579 — APPROVED** [review]
LGTM! Only a couple of minor suggestions.

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      machine {
+        # Virtual machine type, as listed by GCP CLI:
+        # gcloud compute machine-types list --filter="zone:( europe-west3-a )"
+        type = "e2-micro"
+        # Certified image configuration
+        image {
+          type = "certified"
+          java = "latest" # Possible values : 8, 11, 17 or latest
+        }
+        # Storage configuration
```
Comment (tpetillot) [inline content/reference/admin/private_locations/configuration/gcp/index.md:54]: Missing optional `network-interface`

**Review by tpetillot — CHANGES_REQUESTED** [review]

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        disk {
+          # Disk size in Gb (mininum 20Gb)
+          sizeGb = 20
+        }
+      }
+      # GCP project id as returned by GCP CLI:
+      # gcloud projects list
+      project = "<MyProject ID>"
```
Comment (tpetillot) [inline content/reference/admin/private_locations/configuration/gcp/index.md:62]: We don't use "<>" elsewhere in the documentation.

File: content/reference/admin/private_locations/configuration/gcp/index.md (hunk 4 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/gcp/index.md:54]: See https://github.com/gatling/frontline-cloud-doc/pull/167

File: content/reference/admin/private_locations/configuration/gcp/index.md (hunk 1 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/gcp/index.md:69]: Only with java 21 not supported yet

**Pull request merged**